api/autoload/autoload.php

```diff
@@ -0,0 +1,15 @@
+<?php
+class Autoloader
+{
+    public static function loader($class)
+    {
+        $filename ='api/'. $class . '.php';
+        $file =$filename;
+        if (!file_exists($file))
+        {
+            return false;
+        }else{
+        include $file;
+         }
```

Comment:
Code indentation is not the most important thing, but helps us to read and understand code more easily if it is nicely formatted, logically structured, and commented. You should choose a style and be consistent.